### DISCUSSION TITLE
Replacement for PyTorch's `nn.Parameter` in Flax

### DISCUSSION THREAD

**Discussion by tanaymeh** [google/flax#2378] [Q&A] [answered]
Hello everyone!

I was wondering if there is something similar to [PyTorch's](https://pytorch.org/docs/stable/generated/torch.nn.parameter.Parameter.html) `nn.Parameter` class in Flax?

**Comment by marcvanzee** [comment, marked as answer]
Module parameters in Flax are created using `self.param` in a Module. See the [Module parameters](https://flax.readthedocs.io/en/latest/guides/flax_basics.html#module-parameters) section of our "Getting started" tutorial.

**Reply by tanaymeh** [reply]
Thanks a lot!
